websites/K/KissAsian/iframe.ts

```diff
@@ -1,36 +1,11 @@
 const iframe = new iFrame();
 
 iframe.on("UpdateData", async () => {
-	const video: HTMLVideoElement =
-		document.querySelector("#dogevideo_html5_api") ||
-		document.querySelector(
-			"#player > div > div.container.pointer-enabled > video"
-		) ||
-		document.querySelector(
-			"#mediaplayer > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video"
-		) ||
-		document.querySelector("#vid_html5_api") ||
-		document.querySelector("#myElement > div.jw-media.jw-reset > video") ||
-		document.querySelector("#mgvideo > div.vjs-poster") ||
-		document.querySelector("#olvideo_html5_api") ||
-		document.querySelector("#videojs_html5_api") ||
-		document.querySelector(
-			"#myVideo > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video"
-		) ||
-		document.querySelector("#mgvideo_html5_api") ||
-		document.querySelector("#player > div.jw-media.jw-reset > video") ||
-		document.querySelector(
-			"#vstr > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video"
-		) ||
-		document.querySelector(
-			"#player > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video"
-		) ||
-		document.querySelector(
-			"#vstr > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video"
-		) ||
-		document.querySelector("video");
+	const video: HTMLVideoElement = document.querySelector(
+		"video,[class='jw-video jw-reset']"
+	);
 
-	if (video && !isNaN(video.duration)) {
+	if (video && !isNaN(video?.duration)) {
```

Comment:
The question mark seems useless because of the first case in the if...

Reply:
Some videos don't have duration somehow, until a bit later.

Reply:
Yea thats why it starts with if (video)